unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -41,41 +39,44 @@ def set_and_verify_timestamp_domain(sensor, global_time_enabled: bool):
     test.check_equal(frame.get_frame_timestamp_domain(), expected_ts_domain)
```

Comment:
Please use [this](https://github.com/IntelRealSense/librealsense/blob/923253f645e867c75e3c6c30a84127f41d82f3e9/unit-tests/py/rspy/test.py#L327) API to print the frame in case of check failure.

Reply:
```python
    if not test.check_equal(sensor.get_option(rs.option.global_time_enabled), global_time_enabled) or \
            not test.check_equal(frame.get_frame_timestamp_domain(), expected_ts_domain):
        test.info(str(frame.get_profile().stream_type()), frame)
        test.print_info()
```

Reply:
Changed as discussed